Dynamic routes clarification (#8110)
zeit/next.js: packages/next/README.md

</Link>
```

- `href`: the path inside `pages` directory.
- `as`: the path that will be rendered in the browser URL bar.

> As `href` is a filesystem path, it shouldn't change during execution, while you will probably need to change `as`

Reviewer: ```suggestion
As `href` is a filesystem path, it shouldn't change at runtime, instead, you will probably need to change `as`
```

   <Link href="/post?slug=something" as="/post/something">
   ```

>    - `href`: the path inside `pages` directory. Only right side of `?` character could change dynamically during execution, if needed, as left side is a filesystem path.

Reviewer: Let's leave this one as-is since it's not related to dynamic routing.

Author: I agree. Changes removed in https://github.com/zeit/next.js/pull/8110/commits/d217c142acb3a587a555497ad7443ecc12c55367
